### PR TITLE
discharge-wizard: convert .feature to va-summary-box

### PR DESCRIPTION
**Pull request by jamigibbs**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Are you removing, renaming or moving a folder in this PR?
- [x] No, I'm not changing any folders (skip to TeamSites and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

### :warning: TeamSites :warning:
Examples of a TeamSite: https://va.gov/health and https://benefits.va.gov/benefits/. This scenario is also referred to as the "injected" header and footer. You can reach out in the `#sitewide-public-websites` Slack channel for questions.

Did you change site-wide styles, platform utilities or other infrastructure?
- [x] No
- [ ] Yes, and I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#that-sounds-normal-so-whats-the-proxy-all-about) to test the injected header scenario

## Summary

The `.feature` class is deprecated in favor of the [va-summary-box web component](https://design.va.gov/storybook/?path=/docs/uswds-va-summary-box--docs). 

## Related issue(s)

- https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2338

## Testing done

visually inspected locally

## Screenshots

**before**

before

**after**

after

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by Andrew565 — DISMISSED** [review]

**Comment by jilladams** [comment]
@chriskim2311 FYI in case this means we need to change anything in the DUW re-build as subtask pattern to correspond to this PR.

**Comment by laflannery** [comment]
@jamigibbs for additional context, the current H4 is an accessibility violation because the heading order is currently H1 > H4 which skips 2 heading levels. We need to update the h4 to an H2 so no heading levels are skipped

**Comment by jamigibbs** [comment]
> @jamigibbs @laflannery I got a request to change this to a summary box which you did thanks! And to change the heading to
> a h2, can you update the h4s to h2s and this can be fixed for this PR thanks!

> @jamigibbs for additional context, the current H4 is an accessibility violation because the heading order is currently H1 > H4 which skips 2 heading levels. We need to update the h4 to an H2 so no heading levels are skipped

Sure thing! 👍🏼

**Review by chriskim2311 — APPROVED** [review]
LGTM thanks for the changes!

File: src/applications/discharge-wizard/components/gpMinorComponents/OptionalStep.jsx [hunk 1]
```diff
@@ -9,42 +9,44 @@ const OptionalStep = ({ formValues }) => {
...
-            rel="noopener noreferrer"
-            href="https://health.mil/Military-Health-Topics/Conditions-and-Treatments/Physical-Disability/Disability-Evaluation/Physical-Disability-Board-of-Review/PDBR-Application-Process"
-          >
-            Apply for a PBDR review (opens in a new tab)
-          </a>
-          .
-        </p>
+      <section className="vads-u-margin-top--3">
+        <va-summary-box>
+          <h2 slot="headline" className="vads-u-font-size--h4">
```
[the remark this answers is not in the document]
Comment (laflannery) [reply src/applications/discharge-wizard/components/gpMinorComponents/OptionalStep.jsx:14]: Thank you!

**Pull request merged**